panacea/query_client.go

```diff
@@ -316,3 +319,20 @@ func (q QueryClient) GetOracleRegistration(oracleAddr, uniqueID string) (*oracle
 func (q QueryClient) GetLightBlock(height int64) (*tmtypes.LightBlock, error) {
 	return q.lightClient.TrustedLightBlock(height)
 }
+
+func (q QueryClient) GetOracleParamsPublicKey() (*btcec.PublicKey, error) {
+	oraclePubKeyBz, err := q.GetStoreData(context.Background(), paramstypes.StoreKey, append(append([]byte(oracletypes.StoreKey), '/'), oracletypes.KeyOraclePublicKey...))
+	if err != nil {
+		return nil, err
+	}
+	if oraclePubKeyBz == nil {
+		return nil, errors.New("the oracle public key's value is nil")
+	}
```

Comment:
For now, I just put an error handling for nil value of oracle public key when querying oracle params in panacea-core.